docs/platform/concepts/service_backups.rst

```diff
@@ -6,14 +6,18 @@ This article provides information on general rules for handling service backups
 About backups at Aiven
 ----------------------
 
-All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. The backup retention times vary based on the service and the selected service plan. Backups we take for managing the service are not available for download for any service type as they are compressed and encrypted by our management platform.
+All Aiven services, except for Apache Kafka® and M3 Aggregator/Coordinator, have time-based backups that are encrypted and securely stored. The backup retention times vary based on the service and the selected service plan. 
+
+Backups taken by Aiven for managing the service are not available for download for any service type. This is because they are compressed and encrypted by our management platform.
 
 Service power-off/on backup policy
 ------------------------------------
 
-Whenever a service is powered on from a powered-off state, we restore the latest backup available.
+Whenever a service is powered on from a powered-off state, the latest available backup is restored.
+
+Services that have been powered off for more than 180 days are reviewed. A notification email will be sent to you to provide time for taking action before the service and backup are deleted as part of the :doc:`periodic cleanup of powered-off services <../howto/cleanup-powered-off-services>`.
```

Comment:
I guess we dont need to be specific about time here: simple present for a general rule/truth?

A notification email is sent...

Reply:
I think it is worth mentioning the time, 180 days; this could be a question trial users, or prospective customers might have.